ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/SDWAN_TESTS.yml

```diff
@@ -0,0 +1,80 @@
+---
+# Testing sdwan
+wan_mode: sdwan
+
+bgp_as: 65000
+
+sdwan_regions:
+  - name: AVD_Land_West
+    id: 42
+    description: AVD Region
+    zones:
+      - name: danger
+        id: 42
+        description: Danger Zone
+        sites:
+          - name: Site421
+            id: 421
+          - name: Site422
+            id: 422
+  - name: AVD_Land_East
+    id: 43
+    description: AVD Region
+    zones:
+      - name: Roswell
+        id: 51
+        description: Zone 51
+        sites:
+          - name: Site511
+            id: 511
+
+bgp_peer_groups:
+  wan_overlay_peers:
+    password: "htm4AZe9mIQOO1uiMuGgYQ=="
+
+sdwan_pathfinders:
+  # Not testing the overloading
+  - hostname: sdwan-pathfinder
+
+wan_ipsec_profiles:
+  control_plane:
+    shared_key: ABCDEF1234567890
+  data_plane:
+    shared_key: ABCDEF1234567890666
+
+sdwan_zone:
+
+default_node_types:
+  - node_type: wan-controller
+    match_hostnames:
+      - "^sdwan-pathfinder$"
+  - node_type: transit
+    match_hostnames:
+      - "^sdwan-transit$"
+  - node_type: edge
+    match_hostnames:
+      - "^sdwan-edge$"
+
+edge:
```

Comment:
qq: Why not collapse the nodes bit into into sdwan_regions, where we are already defining the sites once?  Avoids repetition.

Just need an extra attribute for role for the sites node there. The looback_ipv4_pool can be defined separately as it is here.